docs/destory_error.rst

```diff
@@ -0,0 +1,5 @@
+==================
+Destroy Error Note
+==================
+
+This is the sample error page
```

Comment:
This needs to be filled in with actual info, also you'll need to put it in index.rst I believe.